src/Microsoft.Identity.Client/Platforms/netcore/OsBrowser/DefaultOsBrowserWebUi.cs

```diff
@@ -87,13 +88,37 @@ public void ValidateRedirectUri(Uri redirectUri)
                         "See https://aka.ms/msal-net-os-browser for details", redirectUri.AbsoluteUri));
             }
 
-            int port = redirectUri.Port;
-            if (port < 1 || port == 80)
+            if (redirectUri.Scheme != Uri.UriSchemeHttp)
```

Comment:
>if (redirectUri.Scheme != Uri.UriSchemeHttp) [](start = 12, length = 44)

why will we not allow https?

Reply:
AAD is not ok with https:\\localhost redirect URIs, i.e. it will not allow https:\\localhost:<any_port>. In my mind this makes sense because the browser is a local process - so what certificates would SSL even use? 

But definitely smth to discuss in the security review.

Reply:
thanks for explaining. should really still be allowed. if I have a cert, I should be able to use it.